requirements/constraints.txt

```diff
@@ -11,8 +11,11 @@
 # Django version 2.0.0 dropped support for python 2.7
 Django<2.0
 
-# TODO: Add contraint explanation
-pylint==1.9.3
-pycodestyle==2.5.0
+# Working version with Python 2.7 and version used in Edx Platform
+edx-opaque-keys==0.4.4
```

Comment:
I think that edx-opaque-keys<2.1.0 also supports python2.7 and the master version of edx-platform uses edx-opaque-keys==2.1.0, so I think that if this constraint is needed the comment should be something like Version used in Ironwood version of edx-platform.

Please, also use as constraint something like

edx-opaque-keys<1.0.0

Reply:
Done! Thanks for the suggestions